packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1680,6 +1682,12 @@ export type DeviceManagerState = {
     unparentedViews: LocalVideoStreamState[];
 };
 
+// @beta
+export interface DevicePermissionPrompts {
```

Comment:
I like having device permission in the name! `Prompts` however isn't quite right - will be block joining a call if the permission is required regardless of any prompt activity.. although they do control if prompts are shown
Perhaps `RequiredUserDevicePermissions` or `DevicePermissionRestrictions`?